agent_issuance/src/credential/aggregate.rs

```diff
@@ -304,12 +369,31 @@ impl Aggregate for Credential {
     }
 }
 
+fn calculate_expiration_timestamp(overwrite: Option<CredentialExpiry>) -> Option<identity_core::common::Timestamp> {
+    #[cfg(feature = "test_utils")]
+    let now = "2010-01-01T00:00:00Z".parse::<chrono::DateTime<chrono::Utc>>().unwrap();
+    #[cfg(not(feature = "test_utils"))]
+    let now = chrono::Utc::now();
+
+    let config = config().credential_expiry.clone();
+
+    match overwrite.unwrap_or(config) {
+        CredentialExpiry::Relative(duration) => {
+            let expires_at = now + duration.to_chrono_at_datetime(now);
```

Comment:
Please use `checked_add` and handle overflows otherwise this can panic:
```rust
#[test]
fn large_relative_expiry_panics_due_to_overflow() {
    let overwrite = Some(CredentialExpiry::Relative(
        iso8601_duration::Duration::from_str("P100000000Y").unwrap(), // <-- eight 0's
    ));

    calculate_expiration_timestamp(overwrite);
    // thread 'credential::aggregate::test' panicked at ~/.cargo/registry/src/index.crates.io-6f17d22bba15001f/chrono-0.4.38/src/datetime/mod.rs:1482:38:
    // `DateTime + TimeDelta` overflowed
}
```